api/models.py

```diff
@@ -145,3 +148,30 @@ class User(AbstractUser):
     USERNAME_FIELD = 'email'
     REQUIRED_FIELDS = ['cohort', 'slack_handle']
     objects = UserManager()
+
+
+class SecurityUser(User):
+    badge_number = models.CharField(max_length=30, unique=True)
+
+    USERNAME_FIELD = 'badge_number'
+    REQUIRED_FIELDS = ['first_name', 'last_name',
+                       'badge_number']
+
+    class Meta:
+        verbose_name = "Security User"
+
+
+@receiver(post_save, sender=User)
+def set_username(sender, instance, *args, **kwargs):
+    if not instance.username:
+        instance.username = instance.email
+        instance.save()
+
+
+@receiver(post_save, sender=SecurityUser)
+def set_email(sender, instance, *args, **kwargs):
+    if not instance.email:
+        user_email = "{}@security.com".format(instance.badge_number)
```

Comment:
i think we should use a domain that's not in use.

Reply:
So back to `example.com` or `andelasecurity.com` ?

Reply:
i'd go with example 

![example](https://user-images.githubusercontent.com/13940254/37658227-1ed40a70-2c5e-11e8-97d7-14d5aac5b017.png)

Reply:
Now there is another thing I have learnt today

Reply:
Let me change this

Reply:
you're welcome 😆